plugins-compat-tester/src/main/java/org/jenkins/tools/test/model/PluginRemoting.java

```diff
@@ -192,12 +192,6 @@ public static void computeScmConnection(PomData pomData) {
         }
 
         oldUrl = transformedConnectionUrl;
-        transformedConnectionUrl = transformedConnectionUrl.replaceAll("((ssh)|(http(s)?))://github\\.com", "git://github.com");
-        if(!oldUrl.equals(transformedConnectionUrl)){
-            pomData.getWarningMessages().add("project.scm.connectionUrl should be accessed in read-only mode (with git:// protocol)");
```

Comment:
Does deleting this not cause a test failure? IIRC there was a unit test for this class. Whatever bug you are fixing, you need to be adding a test case for it.

Reply:
Yes, two tests are failing. I thought this repo had a PR Builder.